demo/data/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@raincatcher/demo-data",
```

Comment:
`@raincatcher-demo/sample-data` - to simplify productization we may need two teams. 
That will help us to differenciate between supported modules and examples.
WDYT?